extensions/vertx-http/deployment/src/main/java/io/quarkus/vertx/http/deployment/webjar/WebJarResultBuildItem.java

```diff
@@ -0,0 +1,46 @@
+package io.quarkus.vertx.http.deployment.webjar;
+
+import java.nio.file.Path;
+import java.util.Map;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+import io.quarkus.maven.dependency.ArtifactKey;
+import io.quarkus.maven.dependency.ResolvedDependency;
+
+public final class WebJarResultBuildItem extends SimpleBuildItem {
```

Comment:
I think I would call it `WebJarResultsBuildItem` plural. You have several and it would be less confusing.